elastic/integrations
# [Azure Logs] Rebrand azure spring cloud logs to azure spring apps logs

- Enhancement

## Proposed commit message
This PR updates the Azure Spring Cloud Logs to Azure Spring Apps Logs as per the change mentioned [here](https://spring.io/blog/2022/05/24/azure-spring-apps-enterprise-is-now-generally-available)



## Checklist


## How to test this PR locally

- Install the Azure Spring Cloud Logs integrations and upgrade the integrations and verify the data is getting loaded in the dashboards.
- Check the older name gets replaced with the newer name in all the places.
- ### Verify the changes in the following places
  - Documentation 
  - Add Integrations page
  - Dashboard title, description, Visualisation title

## Related issues
- Closes #4111 

## Screenshots
<img width="1701" alt="Screenshot 2023-10-13 at 6 19 03 PM" src="https://github.com/elastic/integrations/assets/101238137/b0a0fac7-0966-45a3-bd7d-008c9ed3d6cb">
